bitnami/mxnet/templates/server-statefulset.yml

```diff
@@ -0,0 +1,188 @@
+{{- if eq .Values.mode "distributed" }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "mxnet.fullname" . }}-server
+  labels:
+    app.kubernetes.io/name: {{ include "mxnet.name" . }}
+    helm.sh/chart: {{ include "mxnet.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "server"
+spec:
+  podManagementPolicy: {{ .Values.podManagementPolicy }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "mxnet.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: "server"
+  replicas: {{ .Values.serverCount }}
+  serviceName: {{ template "mxnet.fullname" . }}-headless
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "mxnet.name" . }}
+        helm.sh/chart: {{ include "mxnet.chart" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: "server"
+    spec:
+      {{- include "mxnet.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{ toYaml .Values.server.nodeSelector | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{ toYaml .Values.server.tolerations | nindent 8 }}
+      {{- end }}
+      {{- if .Values.affinity }}
+      affinity: {{ toYaml .Values.server.affinity | nindent 8 }}
+      {{- end }}
+      {{- if .Values.cloneFilesFromGit.enabled }}
+      initContainers:
+      - name: git-clone-repository
+        image: {{ include "git.image" . }}
+        imagePullPolicy: {{ .Values.git.pullPolicy | quote }}
+        command:
+        - /bin/sh
+        - -c
+        - |
+          git clone {{ .Values.cloneFilesFromGit.repository }} /app
+          cd /app
+          git checkout {{ .Values.cloneFilesFromGit.revision }}
```

Comment:
You can do everything on a single command:
```diff
-           git clone {{ .Values.cloneFilesFromGit.repository }} /app
-           cd /app
-           git checkout {{ .Values.cloneFilesFromGit.revision }}
+           git clone {{ .Values.cloneFilesFromGit.repository }} --branch {{ .Values.cloneFilesFromGit.revision }} /app

Reply:
Done